installers/MacInstaller.hs

```diff
@@ -35,10 +42,29 @@ main = do
   copyFile "cardano-node" (dir <> "/cardano-node")
   copyFile "log-config-prod.yaml" (dir <> "/log-config-prod.yaml")
   copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "data/ip-dht-mappings" (dir <> "/ip-dht-mappings")
+  copyFile "build-certificates-unix.sh" (dir <> "/build-certificates-unix.sh")
+  copyFile "tls/ca.conf" (dir <> "/ca.conf")
+  copyFile "tls/server.conf" (dir <> "/server.conf")
 
   -- Rewrite libs paths and bundle them
   _ <- chain dir $ fmap T.pack [dir <> "/cardano-launcher", dir <> "/cardano-node"]
 
+  -- Prepare postinstall script
+  createDirectoryIfMissing False resDir
```

Comment:
(After initially acknowledging the issue, I'm changing my tune, now that I'm clear here).

The `postinstall` can't already be invoked -- this is installer generation time, not run time.

Reading the NSIS documentation, nothing seems to suggest that a `FileOpen`/`FileWrite` to a non-existent directory would succeed:
http://nsis.sourceforge.net/Docs/Chapter4.html#fileinst